### DISCUSSION TITLE
Internationalization - how does _DESCRIPTIONS_ work?

### DISCUSSION THREAD

**Discussion by HuifangYeo** [atoti/atoti#771] [Q&A] [answered]
Hi,

following up on https://github.com/atoti/atoti/discussions/768, the `_FORMATTERS_` only works when I input the correct `Hierarchy` and `Hierarchies` for the level:
```
  "_FORMATTERS_": {
    "[Measures].[NotionalUSD]": "DOUBLE[USD$#,###.000]",
	"[Monte Carlo Data].[AsOfDate].[AsOfDate]": "DATE[dd/MM/yyyy]",
	"[Trade Attributes].[BreakDate].[BreakDate]": "DATE[dd/MM/yyyy]",
	"[Trade Attributes].[MaturityDate].[MaturityDate]": "DATE[dd/MM/yyyy]",
	"[Trade Attributes].[TradeDate].[TradeDate]": "DATE[dd/MM/yyyy]"
  }
```

However, for the `_DESCRIPTIONS_` section, I can input the literal `[Hierarchy].[Hierarchies]` instead of its own `hierarchy` and `hierarchies`:
```
    "[Dimension].[Hierarchies]": "Hiérarchies",
    "[Hierarchy].[Hierarchies].[AsOfDate]": "À la date",
    "[Hierarchy].[Hierarchies].[MaturityDate]": "Date d'échéance",
    "[Hierarchy].[Hierarchies].[TradeDate]": "Date de transaction",
    "[Trade Attributes].[BreakDate].[BreakDate]": "Date de pause",
```

On the UI, I don't see any difference. What does the `_DESCRIPTIONS_` effect and what is the correct syntax that we should have:
`[Trade Attributes].[BreakDate].[BreakDate]` or `[Hierarchy].[Hierarchies].[BreakDate]`?

**Comment by HuifangYeo** [comment]
Similarly, does `__CUBES__` affect the cube name displayed highlighted below?  

![image](https://user-images.githubusercontent.com/35210495/224602007-3ff7a1c5-dc7c-404b-a093-4af4f8509332.png)

**Comment by tibdex** [comment, marked as answer]
Looking at https://activeviam.com/activepivot/latest/docs/configuration/internationalization/#example-of-a-json-locale-file, `_DESCRIPTIONS_` is a map from the default description to the one to the translated one. However, at the moment, Atoti UI doesn't currently show these descriptions (tracked in this [internal ticket](https://activeviam.atlassian.net/browse/UI-8197)).

`_CUBES_` works like you expect but Atoti UI currently renders the `name` and not the `caption`. You could create a `UI` ticket for that one ;).
